packages/grid/src/vaadin-grid-column-mixin.js

```diff
@@ -598,6 +598,14 @@ export const ColumnBaseMixin = (superClass) =>
 
     /** @protected */
     _runRenderer(renderer, cell, model) {
+      if (
+        !(model && model.item && !cell.parentElement.hidden) &&
+        renderer !== this._headerRenderer &&
+        renderer !== this._footerRenderer
+      ) {
```

Comment:
This boolean logic is a bit hard to read. Could break it down into variables.

Reply:
Extracted 2 variables: one for the body cell, and one for the whole condition.

Reply:
Looks good. The other if statement (`if (model && model.item) {`) could now be updated to just check the new `isVisibleBodyCell` variable